acceptance-tests/tests/build.gradle

```diff
@@ -33,6 +35,8 @@ dependencies {
   testImplementation 'org.web3j:besu'
   testImplementation 'tech.pegasys.ethsigner.internal:core'
   testImplementation 'tech.pegasys.ethsigner.internal:file-based'
+
+  testCompile "com.github.tomakehurst:wiremock-jre8-standalone:2.25.1"
```

Comment:
nit: using a "mock" something in AT seems like a bad idea - is this being sued safely/sensibly?

Reply:
It's providing the Enclave responses so we can test the Besu APIs independently.

Reply:
I can go with that